packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/Dataset.java

```diff
@@ -25,16 +27,17 @@ public class Dataset extends ResourceType implements Serializable {
 
 	private String quality;
 
-	@JsonbProperty("temporal_resolution")
+	@JsonProperty("temporal_resolution")
 	private String temporalResolution;
 
-	@JsonbProperty("geospatial_resolution")
+	@JsonProperty("geospatial_resolution")
 	private String geospatialResolution;
 
-	private String annotations;
+	//TODO not an object?
+	private Object annotations;
```

Comment:
Make an issue of it.

Reply:
https://github.com/DARPA-ASKEM/Terarium/issues/818